From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Snow <i@xkun.me>
Date: Sun, 15 May 2022 11:41:03 +0800
Subject: [PATCH] Remove the sys/ prefix for the fcntl include

Serenity does not have the header `fcntl.h` in `/usr/include/sys`.
---
 joe/util/checkwidths.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/joe/util/checkwidths.c b/joe/util/checkwidths.c
index 944faef..e8e3441 100644
--- a/joe/util/checkwidths.c
+++ b/joe/util/checkwidths.c
@@ -14,7 +14,7 @@
 #include <arpa/inet.h>
 #include <stddef.h>
 #include <poll.h>
-#include <sys/fcntl.h>
+#include <fcntl.h>	// the prefix "sys/" is not needed on serenity
 #include <termios.h>
 
 #define TO_CHAR_OK(c) ((char)(c))
